plugin/kubernetes/handler_test.go

```diff
@@ -525,7 +524,8 @@ func (APIConnServeTest) Run()                                      {}
 func (APIConnServeTest) Stop() error                               { return nil }
 func (APIConnServeTest) EpIndexReverse(string) []*object.Endpoints { return nil }
 func (APIConnServeTest) SvcIndexReverse(string) []*object.Service  { return nil }
-func (APIConnServeTest) Modified() int64                           { return time.Now().Unix() }
+func (APIConnServeTest) Modified() int64                           { return 2 }
+func (APIConnServeTest) ExtModified() int64                        { return 2 }
```

Comment:
Reason for the changes?

Reply:
For reference: For testing IXFR and IXFR->AXFR fallback, same thing discussed here: https://github.com/coredns/coredns/pull/3452/files#r417266369